src/lib/transports/splunkHEC.js

```diff
@@ -0,0 +1,36 @@
+import winston from 'winston';
+import formatter from '../formatter';
+
+const https = require('https');
+const fetch = require('@financial-times/n-fetch');
```

Comment:
It looks like n-fetch users n-logger on failed requests, so might get a circular dependency here (or infinite logs of doom).

Maybe better to use https://github.com/matthew-andrews/isomorphic-fetch?

Reply:
Definitely don't want infinite logs of doom, changed!